src/main/groovy/com/inxmail/clicktracker/controller/ApiController.groovy

```diff
@@ -0,0 +1,39 @@
+package com.inxmail.clicktracker.controller
+
+import com.inxmail.clicktracker.entity.PostUrl
+import com.inxmail.clicktracker.entity.Url
```

Comment:
Can you explain your reasoning to use two separate entities? What's the difference between `PostUrl` and `Url`?